src/main/kotlin/no/nav/klage/domain/KlageAnkeInput.kt

```diff
@@ -27,6 +30,9 @@ data class KlageAnkeInput(
     val ytelse: String,
     val vedlegg: List<Vedlegg> = emptyList(),
     var fileContentAsBytes: ByteArray? = null,
+    //deprecated, only used to parse old kafka entries
+    val saksnummer: String? = null,
+    val booleanUserChoices: List<String>? = emptyList(),
```

Comment:
Tenker det burde fremkomme at dette feltet også er deprecated. Kan jo bruke annotasjon for det også sikkert?